apps/transport/lib/jobs/resource_history_job.ex

```diff
@@ -340,13 +340,24 @@ defmodule Transport.Jobs.ResourceHistoryJob do
     %{"content-type" => "application/json"}
     iex> relevant_http_headers(%Req.Response{headers: %{"content-type" => ["application/json", "but-also/csv"]}})
     %{"content-type" => "application/json, but-also/csv"}
```

Comment:
Le test-case HTTP 2 où les headers peuvent être en UTF-8 manque, et révèle un souci si on le crée:

```suggestion

    Does not break UTF-8 headers (which can exist in HTTP 2) including diacritics:

    iex> relevant_http_headers(%Req.Response{headers: %{"content-disposition" => [~S(attachment, filename=é.zip)]}})
    %{"content-disposition" => "attachment, filename=é.zip"}

```

Reply:
Je ne comprends pas ce commentaire.

Reply:
C'est une suggestion d'ajout d'un doctest (qui pourrait casser si j'en crois le commentaire de Thibaut !)

Reply:
Oui c'est bien ça ; les headers HTTP 2 sont par défaut en UTF-8. J'ai proposé d'ajouter ce test car la version précédente du code introduisait un problème, et ce test le montrait.

Reply:
(et je vois que le test a été ajouté dans les versions suivantes, donc on est bon là dessus si je lis bien!)

Reply:
> (et je vois que le test a été ajouté dans les versions suivantes, donc on est bon là dessus si je lis bien!)

je l'ai repris tel quel pour être tranquille